src/routes/index.js

```diff
@@ -1,8 +1,8 @@
 import express from 'express';
-import apiRouter from './api';
+// import apiRouter from './api';
 
 const index = express.Router();
 
-index.use('/api', apiRouter);
+// index.use('/api', apiRouter);
```

Comment:
@chibytez is there a reason we are commenting out these lines (Line 2 and 6)?

Reply:
we don't have the api yet